universal-login-relayer/knexfile.js

```diff
@@ -0,0 +1,45 @@
+// Update with your config settings.
+
+module.exports = {
+
+  development: {
+    client: 'postgresql',
+    connection: {
+      database: 'universal_login_relayer_development',
+      user:     'postgres',
+      password: 'postgres'
+    },
+    migrations: {
+      tableName: 'knex_migrations'
+    }
+  },
+
+  test: {
+    client: 'postgresql',
+    connection: {
+      database: 'universal_login_relayer_test',
+      user:     'postgres',
+      password: ''
```

Comment:
I guess the password for test and development should be same (as both will be called from localhost)

Reply:
✅